content/collections/experiment_troubleshooting/en/new-experiment-run.md

```diff
@@ -0,0 +1,77 @@
+---
+id: 71dedb3e-eb17-4ed1-9df0-bcbc78a2e6a2
+blueprint: experiment_troubleshooting
+title: 'New Experiment Run'
+author: 0c3a318b-936a-4cbd-8fdf-771a90c297f0
+source: 'https://www.docs.developers.amplitude.com/experiment/guides/troubleshooting/restarting-experiments/'
+updated_by: 0c3a318b-936a-4cbd-8fdf-771a90c297f0
+updated_at: 1717179968
+---
+Creating a new run of an existing experiment can be useful if you had instrumentation issues that affected data quality, and you've since fixed them. When you create a new run, you exclude any previous user data from the monitoring and analysis of your experiment.
+
+## Prerequisites
+
+There are two primary requirements to be able to crate a new run of your experiment.
+
+1. Your experiment must use Amplitude's default exposure tracking.
+2. Your client SDK version must support experiment restarts.
+
+| SDK          | Minimum version |
+| ------------ | --------------- |
+| JavaScript   | v1.8.0          |
```

Comment:
Sorry one more small change, we realized this version was wrong. Can you update it to `1.10.2`